Dockerfile

```diff
@@ -27,4 +27,8 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
       maintainer="Steve Chan sychan@lbl.gov"
 
 EXPOSE 7058
-ENTRYPOINT [ "/kb/deployment/bin/entrypoint.sh" ]
\ No newline at end of file
+ENTRYPOINT [ "/kb/deployment/bin/dockerize" ]
+CMD [ "-template", "/kb/deployment/conf/.templates/deployment.cfg.templ:$KB_DEPLOYMENT_CONFIG", \
+      "-stdout", "/kb/deployment/services/workspace/glassfish_domain/Workspace/logs/server.log", \
```

Comment:
Sure, but we want to see the status of the server from the console. If the server is sick, it may not even be able to log to syslog, but it would report it in the local logs and console.